nahimanajz/AndelaWayfarerbus
# #167757068 Merging into gh-pages branch

### What does this PR do?
Updating functionality of the template
### Description of Task to be completed?
Currently:

As an admin or a user, I can see the updated template
Expected

User and admin can see the updated template after cloning this repo and clicking on index.html file
STEPS TO PRODUCE

-Cloning repo
-Open UI/folder 
-Open Index.html

### How should this be manually tested?
After cloning the repo you open UI/HTML Folder  you can browse by clicking on index.html to see the changes
### What are the relevant pivotal tracker stories?
[167757068](https://www.pivotaltracker.com/story/show/167757068)
